qemu/tests/virtio_fs_localfs_migration.py

```diff
@@ -0,0 +1,194 @@
+import re
+
+from virttest import error_context, utils_disk, utils_misc, utils_test
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Basic migration test with diffirent cache modes over localfs
+    Steps:
+        1. Create a shared directory on the host and write a file
+        2. Run the virtiofsd daemon on the host with different cache modes
+        3. Boot the source guest with the virtiofs device in step1
+        4. Mount the virtiofs targets inside the guest
+        5. Create a different directory on the host and write a file
+        6. Run the virtiofsd daemon to share the directory in step5
+        7. Boot the target guest with the virtiofs device in step5
+        8. Do migration from the source guest to the target guest
+        9. No error occurs, the virtiofs is mounted automatically and
```

Comment:
@hellohellenmao Do we have steps to check if writing works on the virtiofs shared dir after migration?

Reply:
#4190 covered the writing scenario

Reply:
#4190  is based on NFS, @hellohellenmao what's your opinion? It's just a suggestion, I thought reading and writing are both important for virtiofs file system migration, they are the basic function.

Reply:
Hi @xiagao and @hellohellenmao , based on your inputs, I would like to echo something to the file after migration to test it's writable, thanks.